network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgVpnControllerService.kt

```diff
@@ -39,7 +39,7 @@ interface WgVpnControllerService {
     ): List<EligibleServerInfo>
 }
 
-const val NETP_ENVIRONMENT_URL = "https://staging.netp.duckduckgo.com"
+const val NETP_ENVIRONMENT_URL = "https://controller.netp.duckduckgo.com"
```

Comment:
You need to add this in `WgVpnControllerRequestInterceptor`

Reply:
Having that said, we might need to integrate the `NetPInternalEnvDataStore` here as well especially if we are using a different endpoint. Completely missed this.

Reply:
If we don't do this, the auth token will not be added.